frontend/tests/pages/subscribe/SubscriptionForm.test.tsx

```diff
@@ -42,4 +22,47 @@ describe("SubscriptionForm", () => {
 
     expect(button).toBeInTheDocument();
   });
+
+  it("calls subscribeEmail action on submit", () => {
+    mockSubscribeEmail.mockImplementation(() => Promise.resolve());
+    render(<SubscriptionForm />);
+
+    const button = screen.getByRole("button", { name: "form.button" });
+    button.click();
+
+    expect(mockSubscribeEmail).toHaveBeenCalledWith(
+      { errorMessage: "", validationErrors: {} },
+      expect.any(FormData),
+    );
+  });
+
+  // unclear why, but React server actions are still not working with Jest here
+  // action function is replaced with `javascript:throw new Error('A React form was unexpectedly submitted')`
+  // See also https://github.com/facebook/react/blob/f83903bfcc5a61811bd1b69b14f0ebbac4754462/packages/react-dom-bindings/src/client/ReactDOMComponent.js#L468
+  // - DWS 2-12-25
+  it.skip("shows relevant errors returned by action", () => {
```

Comment:
Thanks for stubbing this out and leaving it, hopefully support for this lands sooner than later.